### PR TITLE
Add Unichain Sepolia Testnet

### PR DESCRIPTION
**Pull request by trianglesphere**
# Adding a New Chain
This PR adds Unichain Sepolia  to the registry.

## Checklist
- [x] I have declared the chain at the appropriate [Superchain Level](../docs/glossary.md#superchain-level-and-rollup-stage).
- [x] I have run `just validate <chain-id>` locally to ensure all local validation checks pass. 
- [x] I have run `just codegen` to ensure that the chainlist and other generated files are up-to-date and include my chain.

### CODE REVIEW

File: chainList.json
Comment (sbvegan) [inline chainList.json:425]: Can a platform engineer reviewer take a look at why this was against `sepolia-dev-0` instead of `sepolia`?

Comment (tynes) [reply chainList.json:425]: I think it should just be `sepolia` but I don't have a ton of context

**Comment by sbvegan** [comment]
@trianglesphere did you specify sepolia-dev-0 in your [environment file](https://github.com/ethereum-optimism/superchain-registry/blob/1744ce0a4d83dd672562e2ddee2bd6715eb9984b/.env.example#L6)?

File: chainList.json
Comment (sbvegan) [reply chainList.json:425]: very weird

Comment (trianglesphere) [reply chainList.json:425]: you're commenting on something other than our chain definition

Comment (sbvegan) [reply chainList.json:425]: Yep, you're right, sorry I'm on mobile. Overall the PR looks good, it looks like there's a test failure because of a missing newline

Comment (sbvegan) [reply chainList.json:425]: We're also addressing the 1s blocktime check

Comment (bitwiseguy) [reply chainList.json:425]: After I can get an approver and get this pr merged, you should be able to rebase and get the blocktime validation checks to pass: https://github.com/ethereum-optimism/superchain-registry/pull/631

File: validation/genesis/genesis-allocs_test.go
Comment (bitwiseguy) [inline validation/genesis/genesis-allocs_test.go:181]: Still checking with governance about how to workaround the new predeploys. Now sure this is how we want to handle it

Comment (tynes) [reply validation/genesis/genesis-allocs_test.go:181]: These are preinstalls, not predeploys. The predeploys live in a particular address space `0x42...xx`

I think this is fine to merge given that its passing and we can ticket some follow up action items once we figure them out

**Review by tynes — APPROVED** [review]

**Review by sbvegan — APPROVED** [review]
This looks fine for now. @bitwiseguy can we please address Ben's request and put the exception in a toml file. We can chat more about the details on slack or in a call.

Note: we still need to fix these merge conflicts, but everything else looks fine.

**Review by bitwiseguy — APPROVED** [review]

**Pull request merged**
